examples/capi-quick-start/aws-cluster-class.yaml

```diff
@@ -25,6 +25,28 @@ spec:
         discoverVariablesExtension: clusterconfigvars.capi-runtime-extensions
         generateExtension: clusterconfigpatch.capi-runtime-extensions
       name: cluster-config
+    - definitions:
```

Comment:
These should be created as external patches.

Reply:
💯 , the cluster was not coming up though without it and this just a stop gap.

Reply:
Yeah I'd be happy to merge in this state. I'd prefer to merge nfd pr though and then release 0.1.0 before getting this in I think